zbench.zig

```diff
@@ -8,6 +8,13 @@ const log = std.log.scoped(.zbench);
 const c = @import("./util/color.zig");
 const format = @import("./util/format.zig");
 
+pub const Config = struct {
+    iterations: u16 = 0,
+    max_iterations: u16 = 16384,
+    budget: u64 = 2e9, // 2 seconds
```

Comment:
Sorry if this is overly nitpicky, but why the name `budget` and not `max_time` or something similar to indicate we are talking about time? Maybe something like `time_budget`?

Reply:
> Sorry if this is overly nitpicky, but why the name `budget` and not `max_time` or something similar to indicate we are talking about time? Maybe something like `time_budget`?

I was about to write a similar comment ;-) I think it's important to first clarify what user-defined boundaries should there be and which one takes precedence if their implications overlap. Then the proper naming might become more obvious.

Reply:
true 👍 will apply that.